[DOCS] Editorial ILM cleanup (#57565)
elastic/elasticsearch: docs/reference/ilm/set-up-lifecycle-policy.asciidoc

[role="xpack"]
[testenv="basic"]
[[set-up-lifecycle-policy]]
== Configure a lifecycle policy [[ilm-policy-definition]]

For {ilm-init} to manage an index, a valid policy 
must be specified in the `index.lifecycle.name` index setting. 

To configure a lifecycle policy for <<index-rollover, rolling indices>>, 
you create the policy and add it to the <<indices-templates, index template>>.

To use a policy to manage an index that doesn't roll over,
you can specify a lifecycle policy when you create it.

{ilm-init} policies are stored in the global cluster state and can be included in snapshots
by setting `include_global_state` to `true` when you <<snapshots-take-snapshot, take the snapshot>>. 
Restoring {ilm-init} policies from a snapshot is all-or-nothing. 

> The entire global state, including all policies, is overwritten when you restore a snapshot.

Reviewer: This is a bit misleading to me, because it's only policies that have the same names that are overwritten, policies that exist locally but not in the snapshot are not removed. Maybe we can clarify it a little bit?

> Restoring {ilm-init} policies from a snapshot is all-or-nothing. 
> The entire global state, including all policies, is overwritten when you restore a snapshot.

Author: ```suggestion
When the snapshot is restored, all of the policies in the global state are restored and any local policies with the same names are overwritten.
```
